acceptance-tests/Makefile.am

```diff
@@ -2425,4 +2445,13 @@ $(CORECLR_PATH)%.exe: $(CORECLR_PATH)%.cs
 test-runner.exe: $(top_srcdir)/mono/tests/test-runner.cs
 	$(MCS) -debug -out:$@ $<
 
-CLEANFILES = $(CORECLR_TESTSI_CS) $(CORECLR_TESTSI_IL) *.dll *.exe *.mdb
+Tests:
+	mkdir Tests
```

Comment:
The issue is the this folder is hardcoded into the test runner in the coreclr. Also, I copied the file from Kumpera's fork (https://github.com/kumpera/coreclr-gc-test), as he has to make some minor changes to make it cross platform. We should see to upstream these changes, as they do not modify the behavior, but only make it correct on multiple platforms.